Add fixes

Ilanaya opened the pull request.

Commit:
Add fixes

zardoy committed:
[wip] migrate to modern eslint config, change format options

zardoy committed:
up pnpm infra

zardoy commented:
<!--
to test:
- [ ] after code changes reload (partial reload) extension output should be still opened (test with console.log)
-->

zardoy committed:
up eslint introduce faster code reload (ext host restart)
fix bugs: return api, no bootstrap config in prod, pass define to esbuild, pass args to exec
todo review changelog
